Build failure:
```
======================================================================
FAIL: test_short_detail_xml (corehq.apps.app_manager.tests.test_case_detail_distance.CaseDetailDistance)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_case_detail_distance.py", line 48, in test_short_detail_xml
    template_xpath
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/util.py", line 26, in assertXmlPartialEqual
    self.assertXmlEqual(expected, actual, normalize=False)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/util.py", line 32, in assertXmlEqual
    _check_shared(expected, actual, LXMLOutputChecker(), "xml")
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/util.py", line 126, in _check_shared
    raise AssertionError(message)
AssertionError: XML mismatch
--- want.xml
+++ got.xml
@@ -12,7 +12,7 @@
     </template>
     <sort direction="ascending" order="1" type="double">
       <text>
-        <xpath function="if(case_name = '', 2147483647, round(distance(case_name, here())))"/>
+        <xpath function="if(case_name = , 2147483647, round(distance(case_name, here())))"/>
       </text>
     </sort>
   </field>
```

--- a/corehq/apps/app_manager/detail_screen.py
+++ b/corehq/apps/app_manager/detail_screen.py
@@ -280,7 +280,7 @@ class TimeAgo(FormattedDetailColumn):
 @register_format_type('distance')
 class Distance(FormattedDetailColumn):
     XPATH_FUNCTION = u"if(here() = '' or {xpath} = '', '', concat(round(distance({xpath}, here()) div 100) div 10, ' km'))"
-    SORT_XPATH_FUNCTION = u'if({xpath} = '', 2147483647, round(distance({xpath}, here())))'
+    SORT_XPATH_FUNCTION = u"if({xpath} = '', 2147483647, round(distance({xpath}, here())))"
     SORT_TYPE = 'double'
 
 

--- a/corehq/apps/app_manager/tests/test_case_detail_distance.py
+++ b/corehq/apps/app_manager/tests/test_case_detail_distance.py
@@ -129,7 +129,7 @@ class CaseDetailDistance(SimpleTestCase, TestXmlMixin):
                     </template>
                     <sort direction="descending" order="1" type="double">
                         <text>
-                            <xpath function="if(gps= '', 2147483647, round(distance(case_name, here())))"/>
+                            <xpath function="if(gps = '', 2147483647, round(distance(case_name, here())))"/>
                         </text>
                     </sort>
                 </field>

--- a/corehq/apps/app_manager/tests/test_case_detail_distance.py
+++ b/corehq/apps/app_manager/tests/test_case_detail_distance.py
@@ -129,7 +129,7 @@ class CaseDetailDistance(SimpleTestCase, TestXmlMixin):
                     </template>
                     <sort direction="descending" order="1" type="double">
                         <text>
-                            <xpath function="if(gps = '', 2147483647, round(distance(case_name, here())))"/>
+                            <xpath function="if(gps = '', 2147483647, round(distance(gps, here())))"/>
                         </text>
                     </sort>
                 </field>

